components/ActivityIndicator/tests/unit/ActivityIndicatorTests.m

```diff
@@ -167,6 +167,49 @@ - (void)testStopsAnimatingWhenHidden {
   XCTAssertFalse(indicator.animating);
 }
 
+- (void)testTraitCollectionDidChangeBlockCalledWhenTraitCollectionChanges {
```

Comment:
Tip: Just write the second test (that verifies the arguments) since that includes the behavior of the first test.

Reply:
Done